src/user/profile/content/PostContent.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from "react";
+import { Button, Modal, Form, Col } from "react-bootstrap";
+import "./PostContent.scss";
+import { connect } from "react-redux";
+import parse from "html-react-parser";
+import { getAllCommentsOfPost } from "../../../actions/commentAction";
+import userIcon2 from '../../../images/userIcon2.jpg'
+
+class PostContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      post: {},
+      commentsLoaded: false,
+      postId: "",
```

Comment:
I think we are not using postId and commentsLoaded anywhere so maybe they can be removed from the state.